Resources/input/fsh/profiles/ResponseBundle.fsh

```diff
@@ -0,0 +1,55 @@
+Profile: GEM_PR_ERP_MEDREQ_ResponseBundle
```

Comment:
Dieses Bundle kann 2 Adressaten oder? Entweder die Pflegeeinrichtung, die das initial abgeschickt hat, oder die Apotheke, die das einlöst. 
Setzt der Arzt sich selbst dann als RequestingOrganization ein oder bräuchte es dann noch ein Entry sowas wie "PrescribingOrganization"

Reply:
> Dieses Bundle kann 2 Adressaten oder? Entweder die Pflegeeinrichtung, die das initial abgeschickt hat, oder die Apotheke, die das einlöst.
korrekt.

> Setzt der Arzt sich selbst dann als RequestingOrganization ein oder bräuchte es dann noch ein Entry sowas wie "PrescribingOrganization"
Die Objekte sollen nicht neu befüllt werden, sondern nur vom Request in den Response übernommen werden. Die Notwendigkeit eine "PrescribingOrganization" vorzusehen, haben wir bisher gesehen. Könnte aber kommen, da der _KBV_PR_FOR_Practitioner_ keine Angabe zur (telefonischen) Erreichbarkeit oder Adresse vorsieht, sollte dies der Fall sein, müsste eine PrescribingOrganization im Responsebundle vom Typ  [KBV_PR_FOR_Organization](https://simplifier.net/packages/kbv.ita.for/1.1.0/files/720093) eingebunden werden.

Lasst uns das nochmal bewerten!

Reply:
Problem hat sich gelöst da, der MessageHeader [KBV_PR_FOR_Organization](https://simplifier.net/packages/kbv.ita.for/1.1.0/files/720093) Ressource im Sender des ResponseBundles vorsieht. Damit ist sichergestellt, dass der Arzt für Rückfragen erreicht werden kann.